source/php/Module/EventForm/views/components/fields/radio.blade.php

```diff
@@ -0,0 +1,16 @@
+@php ($i = 0)
+@foreach($field['options'] as $value => $label)
+    @option([
+        'type' => 'radio',
+        'checked' => $i === 0,
```

Comment:
$1 can be replaced with laravels loop variable "$loop->index" (docs: https://laravel.com/docs/8.x/blade#the-loop-variable)